.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino EOF
```

Comment:
An End of File is a line break at the end of the file, as Manuel suggested. It would be more something like:
```suggestion
* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino

```

The absence of and End of File on github is indicated by this simbol: 
![image](https://user-images.githubusercontent.com/34688953/174878410-e4232658-fecc-434e-a030-6fd1f5556acc.png)